interfaces/HO-Portal/src/assets/i18n/en.json

```diff
@@ -304,7 +304,8 @@
             "whatsappPhoneNumber": {
               "label": "WhatsApp number",
               "explanation": "This value is required for a 'WhatsApp-enabled' FSP only."
-            }
+            },
+            "fspChangeWarning": "Selecting this FSP will delete the following attributes related to the old FSP:"
```

Comment:
It's not super important, but my initial intention with this structure was that all child-elements of the `...edit-person-affected-popup.properties`-object are only "field/column names".
So this message is outside of that scope, so it could get "its own branch" below `edit-person-affected-popup`, maybe.